src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -163,32 +167,67 @@ private ProcessResult tryCreateNewCase(
             return new ProcessResult(emptyList(), singletonList(AWAITING_PAYMENTS_MESSAGE));
         } else {
             List<Long> ids = ccdApi.getCaseRefsByBulkScanCaseReference(exceptionRecord.id, configItem.getService());
-            if (ids.isEmpty()) {
-                return ccdNewCaseCreator.createNewCase(
-                    exceptionRecord,
-                    configItem,
-                    ignoreWarnings,
-                    idamToken,
-                    userId,
-                    exceptionRecordData
-                );
-            } else if (ids.size() == 1) {
-                final Map<String, Object> finalizedExceptionRecordData =
-                    exceptionRecordFinalizer.finalizeExceptionRecord(
-                        exceptionRecordData.getData(),
-                        ids.get(0),
-                        CcdCallbackType.CASE_CREATION
-                    );
-                return new ProcessResult(finalizedExceptionRecordData);
-            } else {
```

Comment:
not sure why is this being removed ? If a case exists then will it not create a new case ?